content/en/security/guide/security-findings-migration.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Migrating to the New Security Findings Data Model
+description: Learn how to migrate to the unified Security Findings query syntax and data model, which consolidates querying across Cloud Security and Code Security findings.
+private: true
+further_reading:
+- link: "/security/cloud_security_management/"
+  tag: "Documentation"
+  text: "Cloud Security"
+- link: "/security/code_security/"
+  tag: "Documentation"
+  text: "Code Security"
+- link: "/security/application_security/"
+  tag: "Documentation"
+  text: "Application Security"
+---
+
+## Overview
+
+The way Security Findings are queried is changing and it may impact your workflows. You notice changes to how queries are constructed in Datadog. A set of [new features](#new-features) exposing that new schema is also released as a part of the upgrade.
```

Comment:
```suggestion
The way queries are constructed so you can find Security Findings in Datadog is changing. While this change comes with a set of [new features](#new-features) exposing that new schema, it may also impact your existing workflows.
```